### PR TITLE
Update sync instructions for `v2.18.1.1` and `v2.24.2-wormchaind` upgrades.

### PR DESCRIPTION
**Pull request by clemensgg**
Closes: #3943

### CODE REVIEW

File: wormchain/syncing.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ git clone https://github.com/wormhole-foundation/wormhole
 
 # checkout latest release (v2.23.0 at time of writing)
```
Comment (nik-suri) [inline wormchain/syncing.md:15]: can you please update this as well to be:

> checkout latest release (v2.24.2-wormchaind at time of writing)

File: wormchain/syncing.md [hunk 2]
```diff
@@ -121,7 +131,7 @@ cp ../mainnet/* config/
 
 ```bash
 ./wormchaind-v2.14.9.6 rollback --home .
-./wormchaind-v2.14.9.6 start --home . --moniker <your-moniker>
+./wormchaind-v2.14.9.6 start --home . --moniker <your-moniker> --halt-height 3151200
```
Comment (nik-suri) [inline wormchain/syncing.md:134]: nice! Didn't realize you could do this.

**Review by nik-suri — APPROVED** [review]
Thank you for the contribution! Shared just one nitpick comment

File: wormchain/syncing.md (hunk 1 above)
Comment (clemensgg) [reply wormchain/syncing.md]: sure @nik-suri, updated the PR

**Comment by clemensgg** [comment]
unfortunately we [ran into another issue syncing on `v2.18.1.1`](https://github.com/wormhole-foundation/wormhole/issues/3943#issuecomment-2143082267), i've marked this as draft for now

**Comment by clemensgg** [comment]
updated instructions to use `--halt-height` when syncing on `v2.18.1`, this is now confirmed to be working.

**Review by evan-gray — APPROVED** [review]
Awesome!

**Review by nik-suri — APPROVED** [review]
Thank you @clemensgg !

**Pull request merged**